cardscan-base/src/main/java/com/getbouncer/cardscan/base/SSDOcrDetect.java

```diff
@@ -27,7 +28,7 @@
     private static float[][] priors = null;
 
     public List<DetectedOcrBox> objectBoxes = new ArrayList<>();
-    boolean hadUnrecoverableException = false;
+    public boolean hadUnrecoverableException = false;
```

Comment:
this is fine for this PR, but in cleaner code, we'd probably want this variable to be only accessible by a getter so it's not mutable outside of this class.